components/centraldashboard-angular/frontend/src/app/types/dashboard-links.ts

```diff
@@ -0,0 +1,25 @@
+export interface DashboardLinks {
+  menuLinks: menuLink[];
+  externalLinks?: any[];
+  quickLinks?: quickLink[];
+  documentationItems?: documentationItem[];
+}
+
+export interface menuLink {
+  type: string;
+  link: string;
+  text: string;
+  icon: string;
+}
+
+export interface quickLink {
+  text: string;
+  desc: string;
+  link: string;
+}
+
+export interface documentationItem {
+  text: string;
+  desc: string;
+  link: string;
+}
```

Comment:
Maybe we can use the same type here. I don't see a need for two different interfaces.